draft-ietf-rtcweb-jsep.xml

```diff
@@ -1976,7 +1977,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>The m= line MUST be followed immediately by a "c=" line,
           as specified in
           <xref target="RFC4566"></xref>, Section 5.7. Again, as no
-          candidates have yet been gathered, the "c=" line must contain
+          candidates have yet been gathered and the pre-gathered
+          candidates pool is not used so the "c=" line must contain
```

Comment:
I liked Harald's proposed text, i.e., "given that no candidates are available yet"